lopper/assists/baremetallinker_xlnx.py

```diff
@@ -282,4 +297,9 @@ def xlnx_generate_bm_linker(tgt_node, sdt, options):
         memip_list.insert(0, memip_list.pop(memip_list.index(has_ocm[0])))
     cfd.write("set(TOTAL_MEM_CONTROLLERS %s)\n" % to_cmakelist(memip_list))
     cfd.write(f'set(MEMORY_SECTION "MEMORY\n{{{mem_sec}\n}}")\n')
+    if stack_size:
+        cfd.write(f'set(STACK_SIZE {stack_size})\n')
```

Comment:
Where are we defining the default size of stack and heap? Is it possible to keep all of them at one place in here?

Reply:
default one we have kept in our linker_gen API...

Reply:
Fixed now with the latest commit added default stack and heap size based on the processor.